packages/connect-examples/expo-playground/app/routes/device-info.tsx

```
@@ -0,0 +1,335 @@
+          isEmpty
+            ? 'bg-muted/5 text-muted-foreground cursor-default'
+            : 'bg-background hover:bg-accent/30 cursor-pointer'
+        }`}
+        title={isEmpty ? `${field} - No data` : `${field}: ${value} (click to copy)`}
+      >
+        <div className="flex justify-between items-start mb-1">
+          <span className="text-xs font-bold text-foreground">
+            {field}
+          </span>
+          {!isEmpty && (
+            <Copy className="w-3 h-3 text-muted-foreground opacity-0 group-hover:opacity-100 transition-opacity" />
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)